Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml

```diff
@@ -0,0 +1,395 @@
+- type: entity
+  abstract: true
+  parent: BaseItem
+  id: PrizeTicketBase
+  description: A prize ticket, ready to be redeemed at a prize counter.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/prizeticket.rsi
+  - type: Item
+    sprite: Objects/Fun/prizeticket.rsi
+    size: Tiny
+  - type: StaticPrice
+    price: 0
+  - type: Tag
+    tags:
+    - Sheet
+    - PrizeTicket
+
+- type: entity
+  parent: PrizeTicketBase
+  id: PrizeTicket
+  name: prize ticket
+  suffix: Full
+  components:
+  - type: Material
+  - type: PhysicalComposition
+    materialComposition:
+      PrizeTicket: 1
+  - type: Stack
+    count: 100
+    stackType: PrizeTicket
+    baseLayer: base
+    layerStates:
+    - ticket
+    - ticket_2
+    - ticket_3
+    - ticket_4
+  - type: Sprite
+    state: ticket_4
+    layers:
+    - state: ticket_4
+      map: ["base"]
+  - type: Item
+    heldPrefix: prize ticket
+  - type: Appearance
+
+- type: entity
+  parent: PrizeTicket
+  id: PrizeTicket10
+  name: prize ticket
+  suffix: 10
+  components:
+  - type: Sprite
+    state: ticket
+  - type: Stack
+    stackType: PrizeTicket
+    count: 10
+
+- type: entity
+  parent: PrizeTicket
+  id: PrizeTicket30
+  name: prize ticket
+  suffix: 30
+  components:
+  - type: Sprite
+    state: ticket
+  - type: Stack
+    stackType: PrizeTicket
+    count: 30
+
+- type: entity
+  parent: PrizeTicket
+  id: PrizeTicket60
+  name: prize ticket
+  suffix: 60
+  components:
+  - type: Sprite
+    state: ticket
+  - type: Stack
+    stackType: PrizeTicket
+    count: 60
+
+- type: entity
+  parent: PrizeTicket
+  id: PrizeTicket1
+  name: prize ticket
+  suffix: Single
+  components:
+  - type: Sprite
+    state: ticket
+  - type: Stack
+    stackType: PrizeTicket
+    count: 1
+
+- type: entity
+  parent: PresentRandom
+  id: PrizeBall
+  name: prize ball
+  description: I wounder whats inside!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/prizeticket.rsi
+    layers:
+      - state: prizeball
+  - type: SpawnItemsOnUse
```

Comment:
This is gross.

Reply:
This is how Presents is done sadly i cannot spawn an ent that is just a random plushie as long i would love to have an ent like this... actully could i?

Is it possible to make an ent on init that just get a random plushie?